natgeo/mortar
# [SYTO-1056] More clean up on Most Read component

https://jira.natgeo.com/browse/SYTO-1056

This story does some minor cleanup on the Most Read component.

To test:

- Pull SYTO-1056
- Run npm test
- Run gulp serve
- Navigate to contentPackages and verify layout
